crates/topos/src/components/regtest/commands/spam.rs

```diff
@@ -67,6 +67,21 @@ pub struct Spam {
     /// If not provided open telemetry will not be used
     #[arg(long, env = "TOPOS_OTLP_SERVICE_NAME")]
     pub otlp_service_name: Option<String>,
+    /// Flag to indicate usage of Kubernetes.
+    #[arg(
+        long,
+        env = "TOPOS_NETWORK_SPAMMER_BENCHMARK",
+        requires = "hosts",
+        requires = "number"
+    )]
+    pub benchmark: bool,
+    /// Template for generating target node entrypoints.
+    /// e.g. `--hosts="http://validator-{N}:1340"`
```

Comment:
What about testing it locally, what pattern should be applied? Add a comment for that

Reply:
The intent is not to run this locally but in a `kubernetes` cluster for now. If ran locally in another `docker` environment, we have the `target-nodes` flag.